File: auto-docs/executables/r/Jet_heatmap.r
library(plotly)

p <- plotly(username='theengineear', key='o9zlr0hy6z')

size <- 50
z <- matrix(0, size, size)
for(r in 1:size) {
    for(c in 1:size) {
        z[r, c] <- sqrt(r*c/size^2)
    }
}
data <- list(
  list(
    z = z, 
    scl = "Jet", 
    type = "heatmap"
  )
)
layout <- list(title = "Jet")

response <- p$plotly(data, kwargs=list(layout=layout, filename="Jet-heatmap", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/python/Jet_heatmap.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')

import numpy as np

size = 50
z = np.zeros((size, size)).tolist()
for r in range(1, size + 1):
    for c in range(1, size + 1):
        z[r-1][c-1] = r+c

data = Data([
    Heatmap(
        z=z,
        scl='Jet'
    )
])
layout = Layout(
    title='Jet'
)
fig = Figure(data=data, layout=layout)

plot_url = py.plot(fig, filename='Jet-heatmap', auto_open=False)